docs/en/contributing/install.md

```diff
@@ -8,33 +8,33 @@ sidebar_label: Install
 
 ## Prerequisites
 
-Before setting up FAST-DNA, install Git and [yarn](https://yarnpkg.com/en/docs/install).
+Before setting up FAST-DNA, install Git and <a href="https://yarnpkg.com/en/docs/install" target="_blank">Yarn</a>.
 
 ## Setup the source repository
 
-Clone the repository, `cd` into the project, install [Lerna](https://github.com/lerna/lerna) and install dependencies:
+Clone the repository, `cd` into the project, install <a href="https://github.com/lerna/lerna" target="_blank">Lerna</a> and install dependencies:
 
 ```shell
 git clone https://github.com/Microsoft/fast-dna.git
 cd fast-dna
 yarn global add lerna
 ```
 
-Install all dependencies:
+Installs all the dependencies:
 
 ```shell
 yarn install
 ```
 
-Prepare the workspaces:
+Builds all the packages:
```

Comment:
Can you elaborate on the need for this change? This now reads as a description of what the command does rather than the declaration of what to do / how to setup the source repo. If the change here is meant simply as documentation of setting up the source repo, I would expect this to follow something along the lines of line 35.

Reply:
Same as above.  I intended it to read as a description and improve the wording and grammar. I didn't do that for lines 35+ because they are sub-commands which happen only beneath the hood.  I would argue they are not necessary on this page.

Reply:
that was my intent. doesn't the command declare what I'm doing?